pepper/test/apps/isBoolVerification_test.cpp

```diff
@@ -0,0 +1,33 @@
+#include <cassert>
+#include <gtest/gtest.h>
+#include <algorithm>
+#include "declarations.h"
+
+struct Private {};
+
+#include <apps/hashing.h>
+#include "util.h"
+
+
+TEST(isBoolVerification, isBoolVerificationWithBools) { 
+    uint32_t length = 2;
+    field254 bits[1] = { 0 };
```

Comment:
This should be `bits[2]`, shouldn't it? Interesting that it doesn't cause problems.